docs/index.html

```diff
@@ -59,40 +59,40 @@ <h1 id="header-1">Features</h1>
                 <p>Now you won't have to anymore. This package allows you to use the latest version of those utility functions in all PHP_CodeSniffer versions from PHPCS 2.6.0 and up.</p>
             </dd>
 
-            <dt>A number of abstract sniff classes which your sniffs can extend.</dt>
+            <dt>Several abstract sniff classes which your sniffs can extend.</dt>
             <dd>
-                <p>These classes take some of the heavy lifting away for a number of frequently occuring sniff types.</p>
+                <p>These classes take some of the heavy lifting away for some frequently occurring sniff types.</p>
             </dd>
 
-            <dt>A collection of static properties for often used token groups.</dt>
+            <dt>A collection of static properties for often-used token groups.</dt>
             <dd>
-                <p>Collections of related tokens as often used and needed for sniffs.</br>
+                <p>Collections of related tokens often used and needed for sniffs.</br>
         These are additional "token groups" to compliment the ones available through the PHPCS native <span><code>PHP_CodeSniffer\Util\Tokens</code></span> class.</p>
             </dd>
 
-            <dt>An ever growing number of utility functions for use with PHP_CodeSniffer.</dt>
+            <dt>An ever-growing number of utility functions for use with PHP_CodeSniffer.</dt>
             <dd>
-                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a docblock, PHPCSUtils got you covered!</p>
+                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a DocBlock, PHPCSUtils has you covered!</p>
```

Comment:
I thought `got` was valid here as well and sounds more fun as it references a colloquial expression. Am I wrong ?

Reply:
By definition, colloquial means it excludes others. Safer to stick with standard English.